lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -339,6 +339,9 @@ public static function get_theme_data( $deprecated = array(), $options = array()
 				);
 			}
 
+			// Disable shadow support for themes without theme.json.
+			$theme_support_data['settings']['shadow'] = false;
```

Comment:
Rather than setting `shadow` to `false`, could we set `settings.shadow.defaultPresets` to `false`?